CMakeLists.txt

```diff
@@ -275,10 +321,10 @@ if ( NOT fletch_ENABLED_Boost)
   list( APPEND KWIVER_LIBRARY_DIRS "${BOOST_ROOT_ARG}/lib")
 endif()
 
-if (fletch_ROOT)
+if (fletch_FOUND)
   # since we have fletch, add include and lib dirs to export
-  list( APPEND KWIVER_INCLUDE_DIRS "${fletch_ROOT}/include")
-  list( APPEND KWIVER_LIBRARY_DIRS "${fletch_ROOT}/lib")
+  list( APPEND KWIVER_INCLUDE_DIRS "${fletch_DIR}/include")
+  list( APPEND KWIVER_LIBRARY_DIRS "${fletch_DIR}/lib")
```

Comment:
on my machine, fletch_ROOT is always empty
I am not sure how it gets set.
I believe it was removed from fletch by Johan, as fletch always puts everything in the install directory now, and nothing goes in the build directory